e2e-playwright/README.md

```diff
@@ -0,0 +1,66 @@
+## Quick Start
+
+To start the development, run:
+
+```
+npm install
+npm run install:browsers (only for the first time)
+npm run test (or npm run test:chrome for browser specific)
+```
+
+where `new-project` is your desired folder. This should result in cloning the project, setting up your repository as
+origin and pushing the starter as your initial commit. The linter and tests will be run, to confirm that everything
+works properly.
+
+## E2E Playwright guide
+
+### How to test locally?
+
+After installation, just run `npm run test` command (having a launched application on a given port is not required,
+playwright will run the application itself locally if it is not open). It will run test in headless mode. In CLI you
+will get listed results, also you can debug it by running command which CLI suggest:
+`npx playwright show-report test-report`. Report will be opened in browser, and you can see which test are
+passing/failing and on which browser. If you enter one specific test, you will see all test steps, and also TRACES which
+are very helpful to debug. If your test is failing, enter the trace and then you can debug step by step and see what
+automatic test did (what it see and what it clicked).
+
+### How to debug test from CI?
+
+1. Go to the **Pipelines** page of the MR
```

Comment:
MR stands for Merge Request? If yes than I'd opt for using `PR` instead of `MR`.
IMHO PR is more common than MR 🙂

Reply:
+1